backend/main_page/views/main_page_view.py

```diff
@@ -2,19 +2,26 @@
 from rest_framework.request import Request
 from rest_framework.response import Response
 from rest_framework.views import APIView
+
 from main_page.views.word_search_view import WordSearch
+from main_page.views.instagram_search_view import InstagramSearch
 
 
 class MainPageView(APIView):
     @staticmethod
     def get(request: Request) -> Response:
         word_searched: Response = WordSearch.as_view()(request)
+        instagram_search: Response = InstagramSearch.as_view()(request)
 
         if word_searched.status_code == 404:
             return word_searched
 
+        if instagram_search.status_code == 404:
+            print("An error has occurred in the InstagramSearch get method.")
```

Comment:
Bro.. Printing an error ;-;. No need to verify the code error.

Reply:
Wut, is that "if" not necessary?